src/batSet.mli

```diff
@@ -121,6 +121,12 @@ sig
       The elements of [s] are presented to [f] in increasing order
       with respect to the ordering over the type of the elements. *)
 
+  val at_exn: int -> t -> elt
+  (** [at_exn i s] returns element at rank [i] in [s].
```

Comment:
You might want to be a bit more explicit about what you mean by `at rank [i]` -- that it is about the ordering.